drivers/sensor/bmi160/emul_bmi160.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_BMI160_EMUL_BMI160_H_
+#define ZEPHYR_DRIVERS_SENSOR_BMI160_EMUL_BMI160_H_
+
+#include <zephyr/drivers/emul.h>
+#include <zephyr/drivers/i2c.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+__maybe_unused static bool emul_bmi160_i2c_is_touching_reg(struct i2c_msg *msgs, int num_msgs,
```

Comment:
Why are these defined in the header file?

Reply:
They're currently used in the bmi160 test suite, but I figured that these are fairly common functions for testing that would be used when verifying bus traffic. Where would you have them instead?

Reply:
I figured they should be defined in the .c file with the other bmi160 functions. If you envision using them in tests that would preclude linking in the .c file, I might recommend having some kind of common/utility .c file that could be linked in, but they're OK here too. You might want to leave a comment about where they're expected to be used.